pkg/cosign/tuf/signer.go

```diff
@@ -36,10 +36,9 @@ type FulcioKeyVal struct {
 func FulcioVerificationKey(email string, issuer string) *Key {
 	keyValBytes, _ := json.Marshal(FulcioKeyVal{Identity: email, Issuer: issuer})
 	return &Key{
-		Type:       KeyTypeFulcio,
-		Scheme:     KeySchemeFulcio,
-		Algorithms: KeyAlgorithms,
```

Comment:
What's `KeyAlgorithms`? Is it okay that we're removing it here?

Reply:
That supplies what hash is used to create the key identifier (the if in  id : keystuff). i think we can remove and fix on sha256, since it's not a security concern, we just need a unique id